From: Jon Bernard <jbernard@debian.org>
Date: Wed, 15 Dec 2010 09:59:38 -0500
Subject: Fix typo in cgred initscript LSB dependency section
To: libcg-devel@lists.sourceforge.net

Here is the relevant lintian information:

    The given init script declares a dependency on a virtual facility
    that is not known to be provided by any init.d script in the
    archive. If the dependency can not be satisfied upon the package's
    installation, insserv will refuse the activation of the init.d
    script.

    Refer to http://wiki.debian.org/LSBInitScripts for details.

I believe this is likely a typo and can be merged upstream.
---
 scripts/init.d/cgred.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/init.d/cgred.in b/scripts/init.d/cgred.in
index d23472c..725d0b3 100644
--- a/scripts/init.d/cgred.in
+++ b/scripts/init.d/cgred.in
@@ -22,7 +22,7 @@
 # pidfile: /var/run/cgred.pid
 #
 ### BEGIN INIT INFO
-# Required-Start:	$local_fs $syslog $cgconfig
+# Required-Start:	$local_fs $syslog cgconfig
 # Required-Stop:	$local_fs $syslog
 # Provides:		cgred
 # Default-Start:        2 3 4 5
-- 
